Drink_Book_PR/DrinkBook/Club.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DrinkBook
+{
+    public abstract class Club
+    {
+        private string name;
+        private int priceOfWhiskey;
+        private int priceOfVodka;
+        private int capacity;
+
+        private string musicPlayed;
+
+        List<User> listOfUsers = new List<User>();
+
+        public string MusicPlayed
+        {
+            get
+            {
+                return musicPlayed;
+            }
+            set
+            {
+                musicPlayed = value;
+            }
+        }
+
+        public string Name
+        {
+            get
+            {
+                return name;
+            }
+            set
+            {
+                name = value;
+            }
+        }
+
+        public int PriceOfWhiskey
+        {
+            get
+            {
+                return priceOfWhiskey;
+            }
+            set
+            {
+                priceOfWhiskey = value;
+            }
+        }
+
+        public int PriceOfVodka
+        {
+            get
+            {
+                return priceOfVodka;
+            }
+            set
+            {
+                priceOfVodka = value;
+            }
+        }
+
+        public int Capacity
+        {
+            get
+            {
+                return capacity;
+            }
+            set
+            {
+                capacity = value;
+            }
+        }
+
+        public Club()
+        {
+            this.name = "";
+            this.priceOfWhiskey = 0;
+            this.priceOfVodka = 0;
+            this.capacity = 0;
+
+        }
+
+        public Club(string name, int priceOfWhiskey, int priceOfVodka, int capacity)
+        {
+            this.name = name;
+            this.priceOfWhiskey = priceOfWhiskey;
```

Comment:
Shouldn't you set the properties values here, instead of the field ones? `this.Name = name`